addOns/selenium/selenium.gradle.kts

```diff
@@ -48,4 +50,5 @@ dependencies {
     zapAddOn("network")
 
     testImplementation(project(":testutils"))
+    testImplementation(libs.log4j.core)
```

Comment:
Did you intend to test logging? Or is this required just for them to run properly?

Reply:
Neither, can remove it though.

Reply:
Removed and clarified a comment.